index.js

```diff
@@ -12,8 +12,16 @@ https://raw.githubusercontent.com/GPII/first-discovery-server/master/LICENSE.txt
 
 var fluid = require("infusion");
 var gpii = fluid.registerNamespace("gpii");
+var path = require("path");
 require("./src/js/firstDiscoveryServer.js");
+require("./src/js/config.js");
+
```

Comment:
This would be a good place to register your module according to the new module API pattern - 

fluid.module.register("first-discovery-server", __dirname, require)

https://github.com/fluid-project/infusion-docs/pull/77/files#diff-d7540666721f8644efd29f0d5fb29613R53

Although we may as well take the opportunity to namespace this module properly in npm since we plan to deploy everything there before long - I suggest a new package name of "gpii-first-discovery-server" but open to alternatives.